commands/user.go

```diff
@@ -101,6 +101,23 @@ var ListUsersCmd = &cobra.Command{
 	Args:    cobra.NoArgs,
 }
 
+var GenerateUserTokenCmd = &cobra.Command{
+	Use:     "generate-token [user] [description]",
```

Comment:
i see other two-word commands use `_`, what is the standard for the two-word commands, `_` or `-`? I personally prefer `-` but just wanted to confirm.


https://github.com/mattermost/mmctl/pull/133/files#diff-dbf54b6a188b305f65470b9f52908c27R63

uses `reset_password`

Reply:
Actually, I'd rather avoid this problem having:

```sh
mmctl token generate
mmctl token list
mmctl token revoke
```

I was thinking on nesting them under `user` (so `mmctl user token generate`) but that would make more sense if we could generate a token for any user. As this is for the authenticated user, I'm happy having just a `token` or an `access-token` subcommand right at the root 1/5